include/arc_utilities/arc_helpers.hpp

```diff
@@ -192,26 +183,16 @@ namespace arc_helpers
         {
             return (current | update_mask);
         }
-        else
-        {
-            update_mask = (~update_mask);
-            return (current & update_mask);
-        }
+        update_mask = (~update_mask);
```

Comment:
Dale doesn't like this

Reply:
If someone forced me to take a side, I'd probably be in favor of the else statement as well.

Reply:
Of all the `else` removals, I'm least passionate about this one.

Reply:
For the record; here's why I don't like this particular change. It changes from an explicit logic to implicit logic in a situation where one of the options is not "exceptional" in any sense. I.e, both logic paths are normal; neither is "the default".